components/brave_rewards/resources/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig",
+  "include": [
```

Comment:
It was the only way I could see to get correct intellisense in Visual Studio Code working for certain types (I think theme properties, but I'm having trouble remembering now). Perhaps the project was otherwise too big unless localized to the specific dir the current file open is in.